server/src/api/v1.py

```diff
@@ -255,6 +255,10 @@ def result_post(job_id, json_data):
     if not check_valid_uuid(job_id):
         abort(400, message="Invalid job_id specified")
 
+    content_length = request.content_length
+    if content_length and content_length >= 16 * 1024 * 1024:
```

Comment:
I think it should be explained where this limit comes from i.e. it's related to some mongodb restriction

Reply:
Good one, sorted.